### PR TITLE
Make postfixlineedit more userfriendly, rename email id string to "Email"

### PR DESCRIPTION
**Pull request by danimo**

### CODE REVIEW

**Comment by danimo** [comment]
@dragotin @ogoffart pls

File: src/gui/wizard/postfixlineedit.cpp [hunk 1]
```diff
@@ -12,122 +12,70 @@
...
-{
-        QValidator::State state = QRegExpValidator::validate(input, pos);
-        if (state == QValidator::Intermediate)
-            state = QValidator::Invalid;
-        return state;
-}
-
-// Begin of URLLineEdit impl
+const int horizontalMargin(4);
+const int verticalMargin(4);
```
Comment (dragotin) [inline src/gui/wizard/postfixlineedit.cpp]: really const 4?

File: src/libsync/theme.h [hunk 2]
```diff
@@ -286,6 +286,9 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
     virtual QString wizardUrlPostfix() const;
 
 
+    virtual QString wizardUrlHint() const;
```
Comment (dragotin) [inline src/libsync/theme.h]: This is missing documentation. Please add.

File: src/libsync/owncloudtheme.cpp [hunk 3]
```diff
@@ -109,8 +109,30 @@ QPixmap ownCloudTheme::wizardHeaderLogo() const
 {
     return QPixmap(hidpiFileName(":/client/theme/colored/wizard_logo.png"));
 }
+
 #endif
 
+QString ownCloudTheme::wizardUrlPostfix() const
+{
+    return ".dattodrive.com";
```
Comment (ogoffart) [inline src/libsync/owncloudtheme.cpp]: leftover from tests?

File: src/gui/wizard/postfixlineedit.cpp (hunk 1 above)
Comment (danimo) [inline src/gui/wizard/postfixlineedit.cpp]: Yes, I'm more than unhappy about magic constants, but I didn't find another way (QLineEdit also uses (slightly different) magic constants internally).

**Comment by danimo** [comment]
@dragotin ?

**Comment by dragotin** [comment]
:+1:

**Pull request merged**